src/common/utils.js

```diff
@@ -100,5 +101,13 @@ module.exports = {
 
   className: function(prefix, clazz, skipDot) {
     return (skipDot ? '' : '.') + prefix + '-' + clazz;
+  },
+
+  escapeHTML: function(str, originalHighlightTags) {
+    var div = document.createElement('div');
+    div.appendChild(document.createTextNode(str));
+    return div.innerHTML
+      .replace(highlightTags.pre, originalHighlightTags.pre)
+      .replace(highlightTags.post, originalHighlightTags.post);
```

Comment:
To replace **all** occurrences, you need to go for the `replace(/.../g, ...)` (regexp) syntax.